### PR TITLE
adds API call for rule deletion

### PR DESCRIPTION
**Pull request by martinheld**

### CODE REVIEW

File: lib/bluekai/rule.rb [hunk 1]
```diff
@@ -161,5 +161,13 @@ def rule_update(rule_id, body)
       body = { partner_id: @partner_id }.merge(body)
       request('PUT', "/Services/WS/classificationRules/#{rule_id}", {}, body)
     end
+    # Public: Deletes a self-classification rule
```
Comment (dziemba) [inline lib/bluekai/rule.rb]: :fearful: No newline!

Comment (martinheld) [inline lib/bluekai/rule.rb]: fixed

**Comment by dziemba** [comment]
:mushroom: , I like the new changes!

**Comment by martinheld** [comment]
nice, letz mergggge

**Pull request merged**